src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java

```diff
@@ -108,22 +108,18 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                     statusValue = "success";
                 } else {
                     LOGGER.error("UpdateRecommendations API request count: {} failed", calCount);
-                    LOGGER.error("UpdateRecommendations API failure response, experiment_name: {} and intervalEndTimeStr : {}",
-                            experiment_name, intervalEndTimeStr);
-                    sendErrorResponse(response, null, kruizeObject.getValidation_data().getErrorCode(), kruizeObject.getValidation_data().getMessage());
+                    sendErrorResponse(response, null, kruizeObject.getValidation_data().getErrorCode(), kruizeObject.getValidation_data().getMessage(), experiment_name, intervalEndTimeStr);
```

Comment:
change this to 

sendErrorResponse(response, null, kruizeObject.getValidation_data(). experiment_name, intervalEndTimeStr);
and in function
you can get errorcode and message from single varaible